Code/Java/CircularQueue

```diff
@@ -0,0 +1,402 @@
+
+/**
+ * Author - Khushee Kapoor
+ * 
```

Comment:
No need to add * in every line while using multiline comments

Reply:
Made the changes.